src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -211,6 +211,19 @@ internal static PackageSpec GetPackageSpec(JsonTextReader jsonReader, string nam
                             }
                         }
                         break;
+
+                    case "SdkAnalysisLevel":
```

Comment:
Similar to the other json spec reader, this code appears to be in the wrong place. There will never be an `SdkAnalysisLevel` property on the "root" `PackageSpec` object, because it's moved to `RestoreMetadata`.